lucene/analysis/common/src/test/org/apache/lucene/analysis/hunspell/TestAllDictionaries.java

```diff
@@ -181,11 +188,16 @@ public void testDictionariesLoadSuccessfully() throws Exception {
       }
     } finally {
       executor.shutdown();
-      executor.awaitTermination(1, TimeUnit.MINUTES);
+      assertTrue(executor.awaitTermination(1, TimeUnit.MINUTES));
```

Comment:
The test runner would also capture and report any escaped threads, even if this falls through.

Reply:
This was yellow-highlighted by the IDE, and it was easier to surrender :)

Reply:
Use vim? :)

Reply:
Oh. I'm too addicted to IntelliJ and its warnings (especially having created some of them) :)